### PR TITLE
Add release notifications in slack

### PR DESCRIPTION
**Pull request by exekias**
This also updates the workflow to explicitly call goreleaser instead of
reacting to the build

### CODE REVIEW

**Comment by sgirones** [comment]
you can remove the if statements on all the build steps :)

**Comment by exekias** [comment]
ouch, wouldn't that make these steps also happen for PRs?

**Comment by exekias** [comment]
Ok I think [def5788](https://github.com/xataio/cli/pull/25/commits/def5788bd78748634a82a0bf2bff7dfa4cf12556) cleaned it a little bit

**Comment by sgirones** [comment]
yeah, I meant during the tests here in this PR, then revert before merging

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -72,12 +72,57 @@ jobs:
       run: |
         go test -failfast -v .
 
+  on-release-start:
+    runs-on: ubuntu-latest
+    needs: build
+    timeout-minutes: 2
+    if: github.ref == 'refs/heads/main'
```
Comment (sgirones) [inline .github/workflows/build.yml:79]: if you remove this line you will be able to tests the whole process before merging. Then you just revert and merge

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -89,3 +134,99 @@ jobs:
     - name: Release new version
       run: |
         echo y | ./scripts/bump_version.sh
+
+  # Tags created by build workflow don't trigger, call goreleaser:
+  publish-release:
+    needs: [build, release]
+    uses: xataio/cli/.github/workflows/goreleaser.yml@slack-notifications
```
Comment (sgirones) [inline .github/workflows/build.yml:141]: you will need to modify this to `main` before merging

**Review by sgirones — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (exekias) [reply .github/workflows/build.yml:141]: The thing is that tests won't pass if I modify it, so I think I will need to change it in a followup PR?

Comment (sgirones) [reply .github/workflows/build.yml:141]: that's an option, yes

**Pull request merged**